.github/workflows/vue-ci.yml

```diff
@@ -0,0 +1,83 @@
+name: SBC-Common-Components Web CI
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "vue/sbc-common-components/**"
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./vue/sbc-common-components
+
+jobs:
+  setup-job:
+    runs-on: ubuntu-20.04
+
+    if: github.repository == 'bcgov/sbc-common-components'
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: "true"
+
+  linting:
+    needs: setup-job
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        node-version: [10.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: |
+          make setup
+      - name: Linting
+        run: |
+          make lint
+
+  testing-coverage:
+    needs: setup-job
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        node-version: [10.x]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: |
+          make setup
+      - name: Test with Jest
+        id: test
+        run: |
+          make test
+      # - name: Upload coverage to Codecov
+      #   uses: codecov/codecov-action@v1
```

Comment:
Not sure if we need codecov setup?

Reply:
You don't need permission for Codecov. Free and Open.